docs/config/_default/markup.toml

```diff
@@ -0,0 +1,29 @@
+defaultMarkdownHandler = "goldmark"
```

Comment:
This is unchanged from the theme file so removing it here should pick it up there.

Reply:
Done!